Lib/test/test_imaplib.py

```diff
@@ -70,7 +70,22 @@ def test_that_Time2Internaldate_returns_a_result(self):
         for t in self.timevalues():
             imaplib.Time2Internaldate(t)
 
+    @staticmethod
+    def _check_port(address='localhost', port=imaplib.IMAP4_PORT):
```

Comment:
The function is only used once. You can just inline the code directly into test_imap4_host_default_value().

It seems like IMAP4 uses '' (empty string) as the default host, not "localhost" which might be a different host depending on the config.